### DISCUSSION TITLE
Special characters

### DISCUSSION THREAD

**Discussion by podaen** [Bodmer/TFT_eSPI#1585] [Q&A - Fonts]
How could I print special characters with drawstring. Take this character is not displayed   °

**Comment by Bodmer** [comment]
That symbol is only included in one font:

https://github.com/Bodmer/TFT_eSPI/blob/master/Fonts/Font16.c#L1-L8

The symbol could be included in a Smooth font.

Alternatively you can use a suitably positioned lower case letter "o" in a smaller font.

**Reply by Bodmer** [reply]
See [example here](https://camo.githubusercontent.com/b65a710352ad254a1a461ac23ec579477042cc0d1ab174ce1ae900cdbc0aa6a0/68747470733a2f2f692e696d6775722e636f6d2f4f526f76774e592e706e67) where I used string "oC" for the celcius marker.

**Comment by podaen** [comment]
The fond looks nice of the example. Which is it? How did you get the string up there?

**Reply by Bodmer** [reply]
This sketch:
https://github.com/Bodmer/OpenWeather/tree/master/examples/TFT_eSPI_OpenWeather_LittleFS

Code:
https://github.com/Bodmer/OpenWeather/blob/master/examples/TFT_eSPI_OpenWeather_LittleFS/TFT_eSPI_OpenWeather_LittleFS.ino#L400-L404

Fonts:
https://github.com/Bodmer/OpenWeather/blob/master/examples/TFT_eSPI_OpenWeather_LittleFS/TFT_eSPI_OpenWeather_LittleFS.ino#L37-L38

Font files:
https://github.com/Bodmer/OpenWeather/tree/master/examples/TFT_eSPI_OpenWeather_LittleFS/data/fonts

See also library examples for Smooth fonts.
